docker/Dockerfile

```diff
@@ -1,10 +1,9 @@
-FROM node:12.15-alpine
+FROM node:lts-alpine
 
 # install docker-compose
-RUN apk update \
-  && apk add --no-cache py-pip python-dev \
-  libffi-dev openssl-dev gcc libc-dev make \
-  && pip install docker-compose
+RUN apk add curl
+RUN curl -L --fail https://github.com/docker/compose/releases/download/1.29.0/run.sh -o /usr/local/bin/docker-compose
```

Comment:
or we can always install latest like so: `https://github.com/docker/compose/releases/latest/download/run.sh`